cmd/feed-dns/main.go

```diff
@@ -67,6 +69,10 @@ func init() {
 		"A label=value pair to attach to metrics pushed to prometheus. Specify multiple times for multiple labels.")
 	flag.IntVar(&awsAPIRetries, "aws-api-retries", defaultAwsAPIRetries,
 		"Number of times a request to the AWS API is retried.")
+	flag.StringVar(&internalCNAME, "internal-cname", "",
+		"the CNAME of the internal facing load-balancer. If specified, external-cname must also be given.")
```

Comment:
Wording is wrong again, it's the hostname of the load balancer. A CNAME is just a type of dns record that aliases to another hostname. I would guess we'd want to support IPs here as well.

Reply:
We don't have a use case for IPs and they work significantly different to everything else because you can multiple IPs for an A record so we skipped implementing them.